src/main/java/ch/njol/skript/structures/StructCommand.java

```diff
@@ -82,10 +82,9 @@ public class StructCommand extends Structure {
 
 	public static final Priority PRIORITY = new Priority(500);
 
-	private static final Pattern
-		COMMAND_PATTERN = Pattern.compile("(?i)^command /?(\\S+)\\s*(\\s+(.+))?$"),
-		ARGUMENT_PATTERN = Pattern.compile("<\\s*(?:([^>]+?)\\s*:\\s*)?(.+?)\\s*(?:=\\s*(" + SkriptParser.wildcard + "))?\\s*>"),
-		DESCRIPTION_PATTERN = Pattern.compile("(?<!\\\\)%-?(.+?)%");
+	private static final Pattern COMMAND_PATTERN = Pattern.compile("(?i)^command\\s+/?(\\S+)\\s*(\\s+(.+))?$");
```

Comment:
```suggestion
	private static final Pattern COMMAND_PATTERN = Pattern.compile("(?i)^command\\s/?(\\S+)\\s*(\\s+(.+))?$");
```

Reply:
The whole issue was about multiple spaces support as it's supported in almost everywhere in Skript
See the realted issue #5414

Reply:
That's why TP and I said to add a fail for the matcher. That's the point. It should Skript error a message. This avoids the exception thrown.

Reply:
That's an extra check, but instead of error while not allow supporting multiple spaces just like almost everywhere in Skript?! that doesn't make sense and with that there is no standard.

Reply:
> That's an extra check, but instead of error while not allow supporting multiple spaces just like almost everywhere in Skript?! that doesn't make sense and with that there is no standard.
> 
> In addition to that, the error doesn't say what's exactly wrong, it just says it's wrong pattern, users will be mislead by an extra space after `command` part while having 20 extra spaces in arguments or elsewhere..

That's not a bad thing. If the pattern doesn't match, they shouldn't be using it that way, the whole point of a matcher. You can add an example to the error to help them understand what the command structure pattern is. It needs one in general.
```java
Skript.error("Invalid command structure pattern. Example: 'command /example <string> [<number = 5>] [<players>]:'");
```

Reply:
> We don't want people being able to define a command like
> 
> ```vbnet
> command                                             /example:
>     trigger:
> ```

Well, users can already do `set                        player's tool to                               diamond sword` so what's the new here? if you want to fix something like this you better do it for everything else, and that will be a good breaking change.

I don't think I saw a programming language disallowing syntaxes when there are extra spaces here and there, this gives freedom to programmers to do what they like with their coding style.

Reply:
The pattern never allowed this in the first place.

Reply:
Because it's probably either a bug or a wrong choice. As said, it's used like this almost everywhere in Skript, we shouldn't just stop it here, if the team agrees on disallowing duplicated spaces in Skript then that's gonna be in another bug breaking change PR.